### ISSUE TITLE
 View > Other Windows > Source Control Explorer does not exist in Visual Studio Community 2022

### ISSUE THREAD

**Issue by DougAtClir** [MicrosoftDocs/azure-devops-docs#13089]
When I tried to do an initial push of a solution from Visual Studio 2022 to a DevOps repository, no files landed in my repository (not one).

The documenation to manually post files to a repo refers to View > Other Windows > Source Control Explorer.  There is no such menu item in Visual Studio 2022.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d9500fc6-7f1d-b785-f30b-0794e70e23fa
* Version Independent ID: 1bc34281-a5b0-746a-8125-6a45d6113863
* Content: [Add files to version control server - Azure Repos](https://learn.microsoft.com/en-us/azure/devops/repos/tfvc/add-files-server?view=azure-devops)
* Content Source: [docs/repos/tfvc/add-files-server.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/repos/tfvc/add-files-server.md)
* Service: **azure-devops-repos**
* Sub-service: **azure-devops-repos-tfvc**
* GitHub Login: @MaximMalygin
* Microsoft Alias: **vijayma**

**Comment by WilliamAntonRohm** [comment]
@DougAtClir -- Doug, thank you for your feedback.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

@vijayma -- Vijay, please look into this issue.

**Comment by DougAtClir** [comment]
The files appeared on the DevOps server about 45 minutes after the initial push to the repository.


  *   I created a new project and repo in the DevOps portal.

  *   I used “Add project to source control” in Visual Studio 2022 for the initial push.

  *   Visual Studio 2022 reported that the operation was successful. Visual Studio 2022 prompted “Create a pull request”?

  *   When I checked the files on the DevOps portal, no files were shown.  There was a message “Add some files” at on the Repo page. I did several refreshes – files were not appearing.

  *   After about 45 minutes, the files appeared in the portal.

***@***.***

[Title: LinkedIn - Description: image of LinkedIn icon]<http://linkedin.com/>[Title: Facebook - Description: image of Facebook icon]<http://facebook.com/> [Title: Twitter - Description: image of Twitter icon] <http://twitter.com/>
Doug Kimzey
Senior Developer and Partner

Email: ***@***.******@***.***>
Mobile: 865-360-9536

2450 E.J. Chapman Drive
Knoxville, TN 37996
www.clirresearch.com<http://www.clirresearch.com>

**Comment by StingyJack** [comment]
@DougAtClir - you can still get to the Source Control Explorer by using the Team Explorer window.   Once you connect to an AzDo instance the button shows up like this. 
![image](https://user-images.githubusercontent.com/15381181/221387121-8a652e5a-e7fe-456b-90e4-69c93a45597e.png)

I do miss that (Alt + V, E, S) shortcut.  If you use Git for version control, the Source Control Explorer is useless as its only for the better of the two version control systems available in Azure DevOps - TFVC.

**Comment by Fabi12345** [comment]
Nope, no such explorer in my visual studio, its either a bug or didnt implement it in VS2022

**Comment by steved0x** [comment]
Thanks for your feedback! We've moved this issue to our internal tracking system as we transition to a [new feedback system](https://aka.ms/contentuserfeedback).
